README-TA.md

```diff
@@ -132,10 +132,9 @@ GitHub இல் பங்களிக்க பொருத்தமான ச
 
 ## திறமூல பங்களிப்பு முயற்சிகள்
 - [Up For Grabs](https://up-for-grabs.net/) - Contains projects with beginner-friendly issues
-- [First Timers Only](https://www.firsttimersonly.com/) - A list of bugs that are labelled "first-timers-only".
 - [First Contributions](https://firstcontributions.github.io/) - உங்கள் முதல் திறமூல பங்களிப்பை 5 நிமிடங்களில் செய்யுங்கள். ஆரம்பகால பங்களிப்புகளுடன் தொடங்க உதவும் ஒரு கருவி மற்றும் பயிற்சி. [Here](https://github.com/firstcontributions/first-contributions) is the GitHub source code for the site and opportunity to make a contribution to the repository itself.
 - [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - திறந்த மூல பங்களிப்புகளை ஊக்குவிக்கும் திட்டம். அக்டோபர் மாதத்தில் குறைந்தது 4 இழுத்தல் கோரிக்கைகளைச் செய்வதன் மூலம் டி-ஷர்ட்கள் மற்றும் ஸ்டிக்கர்கள் போன்ற பரிசுகளைப் பெறுங்கள்.
 - [24 Pull Requests](https://24pullrequests.com) - 24 Pull Requests என்பது டிசம்பர் மாதத்தில் திறந்த மூல ஒத்துழைப்பை மேம்படுத்துவதற்கான ஒரு திட்டமாகும்.
 
 ## உரிமம்
-<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```

Comment:
This can be kept and added back in.

Reply:
Done https://github.com/freeCodeCamp/how-to-contribute-to-open-source/commit/758d7ecc87e7b3ea24e3d7f77a8753146f9194ea